pinot-core/src/test/java/org/apache/pinot/queries/JsonIngestionFromAvroQueriesTest.java

```diff
@@ -0,0 +1,341 @@
+  private static final org.apache.pinot.spi.data.Schema SCHEMA =
+      new org.apache.pinot.spi.data.Schema.SchemaBuilder().addSingleValueDimension(INT_COLUMN, FieldSpec.DataType.INT)
+          .addSingleValueDimension(JSON_COLUMN_1, FieldSpec.DataType.JSON)
+          .addSingleValueDimension(JSON_COLUMN_2, FieldSpec.DataType.JSON)
+          .addSingleValueDimension(JSON_COLUMN_3, FieldSpec.DataType.JSON)
+          .addSingleValueDimension(STRING_COLUMN, FieldSpec.DataType.STRING).build();
+  private static final TableConfig TABLE_CONFIG =
+      new TableConfigBuilder(TableType.OFFLINE).setTableName(RAW_TABLE_NAME).build();
+
+  private IndexSegment _indexSegment;
+  private List<IndexSegment> _indexSegments;
+
+  @Override
+  protected String getFilter() {
+    return "";
+  }
+
+  @Override
+  protected IndexSegment getIndexSegment() {
+    return _indexSegment;
+  }
+
+  @Override
+  protected List<IndexSegment> getIndexSegments() {
+    return _indexSegments;
+  }
+
+  /** @return {@link GenericRow} representing a row in Pinot table. */
+  private static GenericRow createTableRecord(int intValue, String stringValue, Object jsonValue,
+      GenericData.EnumSymbol enumValue, GenericData.Fixed fixedValue) {
+    GenericRow record = new GenericRow();
+    record.putValue(INT_COLUMN, intValue);
+    record.putValue(STRING_COLUMN, stringValue);
+    record.putValue(JSON_COLUMN_1, jsonValue);
+    record.putValue(JSON_COLUMN_2, enumValue);
+    record.putValue(JSON_COLUMN_3, fixedValue);
+    return record;
+  }
+
+  private static Map<String, String> createMapField(Pair<String, String>[] pairs) {
+    Map<String, String> map = new HashMap<>();
+    for (Pair<String, String> pair : pairs) {
+      map.put(pair.getFirst(), pair.getSecond());
+    }
+    return map;
+  }
+
+  private static Schema createRecordSchema() {
+    List<Field> fields = new ArrayList<>();
+    fields.add(new Field("id", create(Type.INT)));
+    fields.add(new Field("name", create(Type.STRING)));
+    return createRecord("record", "doc", JsonIngestionFromAvroQueriesTest.class.getCanonicalName(), false, fields);
+  }
+
+  private static GenericData.Record createRecordField(String k1, int v1, String k2, String v2) {
+    GenericData.Record record = new GenericData.Record(createRecordSchema());
+    record.put(k1, v1);
+    record.put(k2, v2);
+    return record;
+  }
+
+  private static GenericData.EnumSymbol createEnumField(Schema enumSchema, String enumValue) {
+    return new GenericData.EnumSymbol(enumSchema, enumValue);
+  }
+
+  private static GenericData.Fixed createFixedField(Schema fixedSchema, int value) {
+    byte[] bytes = {(byte) (value >> 24), (byte) (value >> 16), (byte) (value >> 8), (byte) value};
+    return new GenericData.Fixed(fixedSchema, bytes);
+  }
+
+  private static void createInputFile()
+      throws IOException {
+    INDEX_DIR.mkdir();
+    Schema avroSchema = Schema.createRecord("eventsRecord", null, null, false);
```

Comment:
(minor) Either remove `Schema.` or remove the static import

Reply:
Done